examples/controlnet/README_sdxl.md

```diff
@@ -89,6 +89,26 @@ To better track our training experiments, we're using the following flags in the
 
 Our experiments were conducted on a single 40GB A100 GPU.
 
+To train this model in a 24GB memory space we will need to lower the resolution to 512 and raise the gradient_accumulation_steps to 8
+
+```
+accelerate launch train_controlnet_sdxl.py\
```

Comment:
wouldn't `gradient_checkpointing` allow going for full 1024 resolution on a 24GB GPU?

Reply:
messed around last night and did not get a successful run https://wandb.ai/mdcurrent/sd_xl_train_controlnet/runs/5fvzz9qw/overview?workspace=user-mdcurrent. I’ll come back later today trying 786 or simmilar

Reply:
was able to get this running with the command now committed but its definitely running close to oom
<img width="643" alt="Screenshot 2023-10-19 at 2 10 07 PM" src="https://github.com/huggingface/diffusers/assets/15005893/d0ea6569-f704-4f7f-937b-8d4dccff01ad">